From 8f670ca5892429d8391a6a19bc7f726cc1f6c143 Mon Sep 17 00:00:00 2001
From: Olof Johansson <olof@lixom.net>
Date: Wed, 21 Nov 2012 02:20:56 -0800
Subject: [PATCH 840/951] Fix clps711x branch entries in
 arm-soc-for-next-contents.txt

Accidentally removed the clps711x/soc branch from the index instead of the /soc2 one.
Fix retroactively instead of rebuilding the branch just to touch this up.

Signed-off-by: Olof Johansson <olof@lixom.net>
---
 arch/arm/arm-soc-for-next-contents.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm/arm-soc-for-next-contents.txt b/arch/arm/arm-soc-for-next-contents.txt
index 1a206d0..d631215 100644
--- a/arch/arm/arm-soc-for-next-contents.txt
+++ b/arch/arm/arm-soc-for-next-contents.txt
@@ -52,6 +52,7 @@ tegra/single-zimage	git://git.kernel.org/pub/scm/linux/kernel/git/swarren/linux-
 
 next/soc:
 ux500/core		git://git.kernel.org/pub/scm/linux/kernel/git/linusw/linux-stericsson.git ux500-core
+clps711x/soc           patches from Alexander Shiyan <shc_work@mail.ru>
 renesas/soc		git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas soc
 calxeda/ecx-2000	git://sources.calxeda.com/kernel/linux.git tags/calxeda-ecx-2000
 integrator/irq		git://git.kernel.org/pub/scm/linux/kernel/git/linusw/linux-integrator.git tags/integrator-irq
@@ -63,8 +64,6 @@ imx/soc			git://git.pengutronix.de/git/imx/linux-2.6.git tags/imx-soc
 				depends on imx/multiplatform
 davinci/soc		git://gitorious.org/linux-davinci/linux-davinci.git tags/davinci-for-v3.8/soc
 +patch <1353347170-10257-1-git-send-email-csd@broadcom.com> "[PATCH V6] Add support for generic BCM SoC chipsets"
-clps711x/soc2		from patches
-				depends on depends/pinctrl-devel
 sunxi/soc		git://github.com/mripard/linux.git tags/tags/sunxi-support-for-3.8
 tegra/soc		git://git.kernel.org/pub/scm/linux/kernel/git/swarren/linux-tegra.git tegra-for-3.8-soc
 tegra/cpuidle		git://git.kernel.org/pub/scm/linux/kernel/git/swarren/linux-tegra.git tegra-for-3.8-cpuidle
-- 
1.8.0

